src/business_logic/fact_state/in_memory_state_reader.rs

```diff
@@ -94,18 +86,14 @@ mod tests {
 
     #[test]
     fn get_contract_state_test() {
-        let mut state_reader =
-            InMemoryStateReader::new(HashMap::new(), DictStorage::new(), DictStorage::new());
+        let mut state_reader = InMemoryStateReader::new(DictStorage::new(), DictStorage::new());
 
         let contract_address = Address(32123.into());
         let contract_state = ContractState::create([1; 32], Felt::new(109), HashMap::new());
 
-        state_reader
-            .global_state_root
-            .insert(contract_address.clone(), [0; 32]);
         state_reader
             .ffc
-            .set_contract_state(&[0; 32], &contract_state);
+            .set_contract_state(&contract_address.to_32_bytes().unwrap(), &contract_state);
```

Comment:
why unwrap ?

Reply:
Because it `.to_32_bytes()` returns a result.
And since it is a test I don't mind using unwrap()